src/tests/components/NewArticle.test.js

```diff
@@ -25,39 +28,39 @@ describe('NewArticle component', () => {
 
   beforeEach(() => {
     mockStore({});
-    wrapper = shallow(<NewArticle {...props} />);
+    wrapper = shallow(<NewArticle
+      {...props}
+      addArticle={addArticle}
+      resetForm={resetForm}
+      handleEditorChange={handleEditorChange}
+      handleChange={handleChange}
```

Comment:
Add all these props to the props const.

Reply:
The constants have been added into a single object `const props {}`